Classes/Models/LabelDetails.swift

```diff
@@ -10,11 +10,13 @@ import Foundation
 
 final class LabelDetails {
 
+    let client: GithubClient
```

Comment:
Is there another way to pass the client around beside attaching it to a model? This puts a **lot** of baggage in these models (the client object is basically a full app session!)

Reply:
Re-tried this in a new PR - didn't do enough homework on how this is handled elsewhere in the app. Thanks for the feedback!